pkg/virt-launcher/virtwrap/agent-poller/agent_poller_test.go

```diff
@@ -99,20 +103,20 @@ var _ = Describe("Qemu agent poller", func() {
 			const interval = 1
 			const expectedExecutions = 3
 
-			commandExecutions := runPollAndCountCommandExecution(interval, expectedExecutions, 0)
+			commandExecutions := runPollAndCountCommandExecution(interval, expectedExecutions, 0, UnitTestPollInitialInterval)
 
 			Expect(commandExecutions).To(Equal(expectedExecutions))
 		})
 
 		It("executes the agent commands based on the minimum interval at initial run", func() {
-			const interval = 30
-			const expectedExecutions = 2
+			const interval = 3
```

Comment:
do we want to add Expect that interval > UnitTestPollInitialInterval ?
else the mechanism won't be trigger, hence not tested actually, and the test will still pass, because it will be always `interval` (for example if interval = UnitTestPollInitialInterval = 1 second)

Reply:
The value 30 is fine, I do not understand why you touched it. The point of the test, as the text says, is to execute the agent commands based on the *minimum* interval, i.e. the initial-interval.
So this "stable" interval is not relevant except for being high enough not to reach it (which now you changed).

Reply:
I think that we should scale also the 30 seconds, once we are changing the interval to 1 second, in this test
but will change it back to 30 seconds.